docs/_docs/configuration.md

```diff
@@ -461,7 +461,11 @@ The default value for `JEKYLL_ENV` is `development`. Therefore if you omit `JEKY
 
 Your environment values can be anything you want (not just `development` or `production`). Some elements you might want to hide in development environments include Disqus comment forms or Google Analytics. Conversely, you might want to expose an "Edit me in GitHub" button in a development environment but not include it in production environments.
 
-By specifying the option in the build command, you avoid having to change values in your configuration files when moving from one environment to another.
```

Comment:
Let's not remove this sentence. `JEKYLL_ENV=production bundle exec jekyll build` is [commonly used](https://github.com/search?l=Shell&o=desc&q=JEKYLL_ENV&s=indexed&type=Code), and does not prevent to use multiple configuration files at the same time.

Reply:
I feel like this is redundant, makes it harder to understand and also kind of wrong.

What is says is "you can move stuff from the config inside the view to then use liquid in your view with inline-if-else to change it, based on the jekyll.environment."

Why should the docs direct people to move config values in the views, if there is a clean way to have multiple config values per env.

Since I now explain this solution one sentence below, I feel like its harder to understand with both options there.

Reply:
@ashmaroli thats very interesting and a strong +1 to re-work this whole paragraph. However, my change only tries to fix the bug which is a missing link between two solutions.